app/src/main/java/helium314/keyboard/latin/settings/Settings.java

```diff
@@ -447,6 +448,18 @@ public static int readVerticalSpaceSwipe(final SharedPreferences prefs) {
         };
     }
 
+    public static int readLanguageSwipeDistance(final SharedPreferences prefs,
+                                                final Resources res) {
+        final int sensitivity = prefs.getInt(
+                PREF_LANGUAGE_SWIPE_DISTANCE, UNDEFINED_PREFERENCE_VALUE_INT);
+        return (sensitivity != UNDEFINED_PREFERENCE_VALUE_INT) ? sensitivity
+                : readDefaultLanguageSwipeDistance(res);
+    }
+
+    public static int readDefaultLanguageSwipeDistance(final Resources res) {
+        return 10;
```

Comment:
Sorry for asking for a change in the same place again, but I'd like to keep the default close to the old value of 4.
5 seems more suitable for that reason, and it's still a noticeable increase over the old value.